### PR TITLE
inventory skeleton

### PR DESCRIPTION
**Pull request by MahletSeyoum**

### CODE REVIEW

File: src/main/java/com/codeninjas/inventoryservice/domain/Inventory.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    @Id
+    @Generated
+    private String inventoryId;
+
+    @NotNull(message = "item code can't be null")
+    @NotBlank(message = "item code can't be blank")
+    private String itemCode;
+
+    @NotNull(message = "quantity can't be null")
+    @NotBlank(message = "quantity can't be blank")
```
Comment (nebyatweb) [inline src/main/java/com/codeninjas/inventoryservice/domain/Inventory.java:27]: Please remove this line
@NotBlank(message = "quantity can't be blank")
this validation will not work for integer

**Review by nebyatweb — APPROVED** [review]
Please check the reviews and make the necessary updates.

File: src/main/java/com/codeninjas/inventoryservice/domain/Inventory.java (hunk 1 above)
Comment (MahletSeyoum) [reply src/main/java/com/codeninjas/inventoryservice/domain/Inventory.java:27]: I have removed it

Comment (nebyatweb) [reply src/main/java/com/codeninjas/inventoryservice/domain/Inventory.java:27]: Please make sure to push the changes.

**Review by nebyatweb — APPROVED** [review]

**Pull request merged**